vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -152,6 +221,21 @@ export const SelectCheckboxes = {
                 }))
             ];
         },
+        _filterCheckboxes() {
+            if (this.filterCheckboxes) return this.filterCheckboxes;
+            return (items, filter) => {
+                return items.filter(item => {
+                    const itemValue = `${item.label || item.value}`.toLowerCase();
+                    filter = filter.toLowerCase();
+                    return itemValue.includes(filter);
+                });
+            };
+        },
+        filteredItems() {
+            return !this.search || !this.searchValue

```

Comment:
no need for the `!this.search` validation

Reply:
use positive operation `return this.searchValue ? this._filterCheckboxes(this._items, this.searchValue) : this._items;`